src/ChipInput.js

```diff
@@ -491,7 +495,7 @@ class ChipInput extends React.Component {
       >
         <div>
           {floatingLabelTextElement}
-          <div style={{ marginTop: floatingLabelText ? 12 : 0 }}>
+          <div style={Object.assign(styles.chipContainer, chipContainerStyle)}>
```

Comment:
This will modify the `styles.chipContainer` object. This should either be `Object.assign({}, styles.chipContainer, chipContainerStyle)` or  `{ ...styles.chipContainer, ...chipContainerStyle }` (I prefer the latter, using the object spread operator).